extensions/ql-vscode/src/databases-ui.ts

```diff
@@ -378,7 +378,17 @@ export class DatabaseUI extends DisposableObject {
 
   handleRemoveOrphanedDatabases = async (): Promise<void> => {
     logger.log('Removing orphaned databases from workspace storage.');
-    let dbDirs =
+    let dbDirs = undefined;
+
+    if (
+      !(await fs.pathExists(this.storagePath) ||
+        !(await fs.stat(this.storagePath)).isDirectory())
+    ) {
+      // ignore a missing or invalid storage directory.
```

Comment:
You had a log message for this before, I suggest keeping that.

Reply:
Sure.